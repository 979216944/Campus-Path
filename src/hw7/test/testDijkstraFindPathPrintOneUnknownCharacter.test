# Test Dijkstra finding a path,
# it prints one "unknown character notFlower" line, if notFlower is not in graph.

# load graph
LoadGraph g flowers.tsv

# find a minimum-cost path with dest node no in the graph
FindPath g notFlower Aconitum