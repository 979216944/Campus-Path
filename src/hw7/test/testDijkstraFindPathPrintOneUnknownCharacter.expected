# Test Dijkstra finding a path,
# it prints one "unknown character notFlower" line, if notFlower is not in graph.

# load graph
loaded graph g

# find a minimum-cost path with dest node no in the graph
unknown character notFlower